dotcom-rendering/src/components/FlexibleGeneral.tsx

```diff
@@ -194,21 +194,25 @@ type BoostedCardProperties = {
 	headlineSizeOnMobile: SmallHeadlineSize;
 	headlineSizeOnTablet: SmallHeadlineSize;
 	imageSize: ImageSizeType;
+	useBoostedFontSizes: boolean;
 };
 
 /**
  * Boosting a standard card will affect the layout and style of the card. This function will determine the properties of the card based on the boost level.
+ * Megaboosted cards will make use of the boosted font sizes specified in card headline.
+ * Boosted cards will use the regular font sizes, to enable the correct headline size on mobile.
  */
 const decideCardProperties = (
 	boostLevel: BoostLevel = 'boost',
```

Comment:
This is an exisiting issue but we don't actually need to access the full BoostLevel type here as these cards can only be 'boost' or  'megaboost' so perhaps we should look to tighten the type here using Pick<BoostLevel, 'boost','megaboost'>